.github/workflows/reusable-test.yml

```diff
@@ -74,38 +74,71 @@ jobs:
   unit:
     needs: setup-turbo
     runs-on: ubuntu-latest
+    # Limit concurrent test jobs to avoid hitting GitHub Actions limits
```

Comment:
The other two jobs in this file `check` and `extensions` and `integration-tests` are using the following container properties. Should we add here as well?

```
    container:
      image: ghcr.io/yldio/asap-hub/node-python-sq:86a189edc900d4e1afdcf3935c697292f69d409b
      credentials:
        username: ${{ github.actor }}
        password: ${{ secrets.GITHUB_TOKEN }}
```